src/main/java/org/spongepowered/api/entity/living/player/User.java

```diff
@@ -32,14 +31,17 @@
 import org.spongepowered.api.entity.ArmorEquipable;
 import org.spongepowered.api.entity.Tamer;
 import org.spongepowered.api.item.inventory.Carrier;
+import org.spongepowered.api.profile.GameProfile;
 import org.spongepowered.api.service.permission.Subject;
 import org.spongepowered.api.util.Identifiable;
 
 import java.util.Optional;
 
 /**
- * A User is the data usually associated with a Player that is persisted across server restarts.
- * This is in contrast to Player which represents the ingame entity associated with an online User.
+ * A User is the data usually associated with a Player that is persisted across
```

Comment:
Thats not the point, this is a PR for changes to AI not fixing checkstyle. Keep it focused and remove this change.